language/analyze/analyze.go

```diff
@@ -132,6 +132,11 @@ func classifyText(ctx context.Context, client *language.Client, text string) (*l
 			},
 			Type: languagepb.Document_PLAIN_TEXT,
 		},
+		ClassificationModelOptions: &languagepb.ClassificationModelOptions{
+			ModelType: &language_pb.ClassificationModelOptions_V2Model{
+				ContentCategoriesVersion: languagepb.ClassificationModelOptions_V2Model_V2,
+			},
+		},
```

Comment:
I pulled the branch to see if I could get it working locally, and I think I did. Here's what you should do:

1. In `language` dir, run: `go get google.golang.org/genproto`.
   You should see output like: `go: upgraded google.golang.org/genproto v0.0.0-20220201184016-50beb8ab5c44 => v0.0.0-20220927151529-dcaddaf36704`
1. Run `go mod tidy`.
1. Update your changes to:
   ```go
		ClassificationModelOptions: &languagepb.ClassificationModelOptions{
			ModelType: &languagepb.ClassificationModelOptions_V2Model_{
				V2Model: &languagepb.ClassificationModelOptions_V2Model{
					ContentCategoriesVersion: languagepb.ClassificationModelOptions_V2Model_V2,
				},
			},
   ```